pinot-common/src/main/java/org/apache/pinot/common/utils/request/RequestUtils.java

```diff
@@ -48,6 +48,14 @@ public class RequestUtils {
   private RequestUtils() {
   }
 
+  public static SqlNodeAndOptions parseQuery(String query)
+          throws SqlCompilationException {
+    long parserStartTimeNs = System.nanoTime();
+    SqlNodeAndOptions sqlNodeAndOptions = CalciteSqlParser.compileToSqlNodeAndOptions(query);
+    sqlNodeAndOptions.setParseTimeNs(System.nanoTime() - parserStartTimeNs);
+    return sqlNodeAndOptions;
+  }
```

Comment:
use a `private static final JsonNode EMPTY_OBJECT_NODE = new ObjectMapper().createObjectNode();
```suggestion
    return parserQuery(query, EMPTY_OBJECT_NODE);
  }
```